### DISCUSSION TITLE
Welcome to scoop Discussions!

### DISCUSSION THREAD

**Discussion by rasa** [ScoopInstaller/Scoop#4482] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rashil2000** [comment]
Hey folks!

I'm an undergrad student and I ♥ Scoop!

I'm hoping to assist as much as I can to bring Scoop back on track and help maintain it regularly.

**Reply by deevus** [reply]
It was when I was an undergrad in CS that I started working on Scoop years ago. Scoop is absolutely perfect for engineering/CS students who need an easy way to get command line utilities for development or research. 

I wonder... would any unis be willing to sponsor the project? :thinking:

**Reply by rashil2000** [reply]
>...would any unis be willing to sponsor the project? 

I don't think my uni does...
